src/matchers/lang/cfamily/CFamilyLangHelper.ts

```diff
@@ -0,0 +1,76 @@
+
+import { LangHelper } from "../LangHelper";
+import { JavaContentStateMachine } from "./java/JavaContentStateMachine";
+import { DoubleString, Normal, SlashSlashComment, SlashStarComment } from "./States";
+
+export class CFamilyLangHelper implements LangHelper {
+
+    /**
+     * Strip C/C++ comments
+     * @param source
+     */
+    public stripComments(source: string) {
+        let stripped = "";
+        const sm = new JavaContentStateMachine();
```

Comment:
The naming of this class is a bit odd here, couldn't it be `CFamilyContentStateMachine`?